src/test/end-to-end/codecept.conf.js

```diff
@@ -22,7 +22,7 @@ exports.config = {
                     'height': 960
                 },
                 args: [
-                    '--headless',
+                    //'--headless',
```

Comment:
Not sure if this was changed just for testing locally, do we want to run the tests without headless?